src/main/java/com/github/mauricioaniche/ck/CKMethodResult.java

```diff
@@ -33,8 +35,16 @@
 	private int logStatementsQty;
 	private boolean hasJavadoc;
 
-	public CKMethodResult(String methodName, boolean isConstructor, int modifiers) {
+	//All direct invocations of methods
+	private Set<String> methodInvocations;
+	//All direct invocations of methods of the same class
+	private Set<String> methodInvocationsLocal;
+	//All indirect invocations of methods of the same class
+	private Set<String> methodInvocationsIndirectLocal;
```

Comment:
I can see you need to store the method invocations in sets here, but you are missing a method that calculates the LCC and TCC, right (maybe even as a method directly in `CKMethodResult`, as you have all the data you need, I suppose?(? Or is this something you will do in a different PR?

Reply:
I will do it in a different PR, because I also need the class-level variable accessed by each method.
Currently, I am preparing a second PR for that one. And the actual calculation comes in another one.